packages/babel-parser/test/fixtures/es2015/uncategorised/359/output.json

```diff
@@ -2,7 +2,7 @@
   "type": "File",
   "start":0,"end":20,"loc":{"start":{"line":1,"column":0},"end":{"line":1,"column":20}},
   "errors": [
-    "SyntaxError: Unexpected reserved word 'await'. (1:6)"
+    "SyntaxError: Can not use 'await' as identifier inside an async function. (1:6)"
```

Comment:
The input code is `const await = foo();`. As reported on twitter by @KFlash, this error message is confusing (https://twitter.com/buzyescayadev/status/1398240006670454784).

I will open a follow-up PR to fix it, to keep this one as only switching the plugin on by default.

Reply:
We could just change the error message to "... async function or top level module".

Reply:
@JLHwung then what will be the error message when you are parsing in an non-async func body or arrow? Same error?

Reply:
Mh right, I can do it here.

Reply:
The token is the one at (1:6): any editor, or the Babel cli, or Babel integrations will point to the `=`.

Reply:
Maybe we can always throw `Unexpected reserved word 'await'.` in modules (since the reason we throw is that it's reserved), and keep throwing the current error in async functions inside scripts (`await` is not reserved there, but it's special-cased to be illegal).